From 0475a8a61d934437ab2b13fadc195756adac5f0d Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Wed, 15 Jun 2022 22:40:10 +0300
Subject: [PATCH] [SW-0] gaudi2: fix style and readability

Several style and readability fixes in ppboot_bringup_preboot()
function.
No functional changes.

Change-Id: I2ec282588a3ecdc7d75f2a310d00883e67fca823
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/arch/gaudi2/arc/common.c b/arch/gaudi2/arc/common.c
index 5b89433..77039e6 100644
--- a/arch/gaudi2/arc/common.c
+++ b/arch/gaudi2/arc/common.c
@@ -519,25 +519,26 @@
 		return ret;
 
 	for (int i = 1; i < num_cycles; i++) {
-		copy_size = (i == (num_cycles-1)) ? (payload_size - MAX_DMA_SIZE*i + CRC_SIZE) :
+		copy_size = (i == num_cycles - 1) ?
+				(payload_size - MAX_DMA_SIZE * i + CRC_SIZE) :
 				(MAX_DMA_SIZE + CRC_SIZE);
 
-		ret = ppboot_dma_copy(copy_size, (flash_offset + OFFSET_64KB * i),
-					ARC1_ICCM_FULL_BASE, SPI_DMA_DST_HBW);
+		ret = ppboot_dma_copy(copy_size, flash_offset + OFFSET_64KB * i,
+				      ARC1_ICCM_FULL_BASE, SPI_DMA_DST_HBW);
 		if (ret)
 			return ARCBOOT_BRINGUP_ERROR;
+
 		memcpy((void *)(SDRAM_BASE + MAX_DMA_SIZE * i),
 		       (void *)ARC_ICCM_BASE, copy_size - CRC_SIZE);
 	}
 
 	if (num_cycles)
-		image_size -= (CRC_SIZE * (num_cycles-1));
+		image_size -= CRC_SIZE * (num_cycles - 1);
 
 	image_size -= SECUREBOOT_IMG_HEAD_FOOTER;
 	if (!preboot_security_fuse_override()) {
-		if (secureboot_verify_image(image_size,
-					    (SDRAM_BASE + IMG_LEN_SIZE))
-		    == SECUREBOOT_VERIFY_FAILED)
+		ret = secureboot_verify_image(image_size, SDRAM_BASE + IMG_LEN_SIZE);
+		if (ret == SECUREBOOT_VERIFY_FAILED)
 			return ARCBOOT_BRINGUP_ERROR;
 	}
 
